src/rsa/bigint.rs

```diff
@@ -218,6 +218,21 @@ pub struct Modulus<M> {
     ring: PhantomData<M>,
 }
 
+impl<M> Modulus<M> {
```

Comment:
Why would we not allow this? JWK also has a private key mapping: https://tools.ietf.org/html/rfc7517#appendix-A.2